### PR TITLE
Adding Empty C# and Node templates

### PR DESCRIPTION
**Pull request by soninaren**

### CODE REVIEW

**Comment by soninaren** [comment]
For #29

**Comment by soninaren** [comment]
@alrod to review if this will cause portal bugs

File: Templates/Empty-CSharp/function.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (alrod) [inline Templates/Empty-CSharp/function.json]: Why does not add "disabled": false ?

Comment (soninaren) [inline Templates/Empty-CSharp/function.json]: being empty, wanted to leave the option to the user to disable or enable it

**Comment by soninaren** [comment]
Talked to @alrod offline. He is okay merging this PR

File: Templates/Empty-CSharp/function.json (hunk 1 above)
Comment (davidebbo) [inline Templates/Empty-CSharp/function.json]: I think setting it explicitly to the default value makes it more usable, without taking anything away from it

**Pull request merged**